airbyte-integrations/connectors/source-dynamodb/metadata.yaml

```diff
@@ -5,7 +5,7 @@ data:
   connectorSubtype: api
   connectorType: source
   definitionId: 50401137-8871-4c5a-abb7-1f5fda35545a
-  dockerImageTag: 0.3.2
+  dockerImageTag: 0.3.3
```

Comment:
Can you bump the doc changelog entry too? Then we're going to be able to run CI integration tests to validate your change.

Reply:
done